### DISCUSSION TITLE
Errors with new Installation - 2024.9.3

### DISCUSSION THREAD

**Discussion by a357281** [TypQxQ/Sigenergy-Home-Assistant-Integration#15] [General]
Receiving some errors on a new installation

Logger: homeassistant.config
Source: config.py:950
First occurred: 12:45:11 (16 occurrences)
Last logged: 12:45:11

Setup of package 'input_select' at packages/modbus_sigenergy.yaml, line 2461 failed: Integration 'set_sigen_indenpendent_phase_power_control' not found.
Setup of package 'input_text' at packages/modbus_sigenergy.yaml, line 2503 failed: Integration 'set_sigen_ess_max_charging_limit' not found.
Setup of package 'input_text' at packages/modbus_sigenergy.yaml, line 2503 failed: Integration 'set_sigen_ess_max_discharging_limit' not found.
Setup of package 'input_text' at packages/modbus_sigenergy.yaml, line 2503 failed: Integration 'set_sigen_pv_max_power_limit' not found.
Setup of package 'automation' at packages/modbus_sigenergy.yaml, line 2539 failed: Invalid package definition 'automation': expected a dictionary. Package will not be initialized

**Comment by TypQxQ** [comment]
Could be some corruption in the file you have.  I have it running on two systems with 2024.9.3

**Comment by TypQxQ** [comment]
Oups, I was still running 2024.9.2
In process of upgrading :D

**Comment by TypQxQ** [comment]
Can confirm no errors on 2024.9.3.
![image](https://github.com/user-attachments/assets/63035404-f5d4-42f8-8da4-91fdd9913e2f)
![image](https://github.com/user-attachments/assets/92ac40b8-10f3-4c49-9138-9681581e0474)

**Comment by a357281** [comment]
strange - it feels like I have missed a step.
Fresh install (docker) of HA - have created new packages dir. - added lines to secrets.yml - dropped modbus_sigenergy.yaml into the packages dir - updated configurations.yml - it can obviously see the modbus_sigenergy.yaml as it reads the file and errors on the lines in the file... - will carry on troubleshooting am new to HA, also it is running on docker but will persevere and update if I suss it out - thanks for confirming version compatability also.

**Comment by a357281** [comment]
found the issue - the configuration yaml - doesnt like "merge_named" for packages - it must be just "named"

**Comment by TypQxQ** [comment]
Good to know. Happy you found the problem!
I'll close this now as resolved 😄
